src/ariel-os-nrf/src/lib.rs

```diff
@@ -55,7 +55,14 @@ ariel_os_embassy_common::executor_swi!(SWI0);
 ariel_os_embassy_common::executor_swi!(EGU0_SWI0);
 
 #[cfg(feature = "executor-interrupt")]
-#[cfg(any(context = "nrf53", context = "nrf91"))]
+#[cfg(context = "nrf5340-net")]
```

Comment:
this seems unrelated

Reply:
EGU0 is used by `nrf-sdc` for ble on nrf5340. There was no conflict before because BLE on nRF MCUs only used the thread executor.
Now we need to change the SWI for BLE to still work on nrf5340.

Reply:
can we just default to using EGU1 for the nrfs? less special cases ...